lib/hangman.rb

```diff
@@ -0,0 +1,54 @@
+class Word
+
+  attr_reader :incorrect_guesses, :correct_guesses, :user_guesses
+
+  def initialize(word)
+    @word = word
+    # @user_guesses = [] # not needed?
```

Comment:
Agreed that it's probably not needed. Also, I try to make sure I get rid of comments before committing :)

Reply:
Thanks for the tip :)